.circleci/config.yml

```diff
@@ -4,6 +4,7 @@ orbs:
   go: circleci/go@1.8.0
   node: circleci/node@5.2.0
   slack: circleci/slack@4.10.1
+  rust: circleci/rust@1.6.1
```

Comment:
Is this the rust version, or just the package version? 1.6.1 super old compiler

Reply:
Just the orb version - latest publish was december 2023, so should be recent enough.
See: https://circleci.com/developer/orbs/orb/circleci/rust